docs.kosli.com/content/getting_started/deployments.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: "Part 8: Deployments"
+title: "Part 8: Deployment Expectations"
 bookCollapseSection: false
 weight: 270
 ---
-# Part 8: Deployments
+# Part 8: Deployment Expectations
 
-The last important piece of information, when it comes to artifacts are deployments. Whenever you (likely with the use of your CI system) deploy an artifact to an environment you should record that information to Kosli. So when you check the status of your environments you know not only what is running there, but also how did it get there. It's an easy way of detecting a manual change was introduced.
+The last important piece of information, when it comes to artifacts are deployments. Whenever you (likely with the use of your CI system) deploy an artifact to an environment you should record that information to Kosli.
```

Comment:
> record that information to Kosli
 Should this be "record that information in Kosli" or "report that information to Kosli"?